cmd/otelcol/config/collector/full_config_linux.yaml

```diff
@@ -167,6 +167,24 @@ receivers:
   fluentforward:
     endpoint: 0.0.0.0:8006
 
+  # Enables scripted inputs receiver
+  # Full configuration here: https://github.com/signalfx/splunk-otel-collector/blob/main/internal/receiver/scriptedinputsreceiver/README.md
+  scripted_inputs/cpu:
```

Comment:
I think we can add a single example since this is for the receiver as a component option and not for documenting its permutations.